selfdrive/car/fw_versions.py

```diff
@@ -203,19 +204,29 @@ def get_fw_versions_ordered(logcan, sendcan, ecu_rx_addrs, timeout=0.1, num_pand
 
   all_car_fw = []
   brand_matches = get_brand_ecu_matches(ecu_rx_addrs)
+  matched_brand: Optional[str] = None
 
   for brand in sorted(brand_matches, key=lambda b: len(brand_matches[b]), reverse=True):
     car_fw = get_fw_versions(logcan, sendcan, query_brand=brand, timeout=timeout, num_pandas=num_pandas, debug=debug, progress=progress)
     all_car_fw.extend(car_fw)
     # Try to match using FW returned from this brand only
     matches = match_fw_to_car_exact(build_fw_dict(car_fw))
     if len(matches) == 1:
+      matched_brand = brand
       break
 
+  # Do non-OBD queries for matched brand, or all if no match is found
+  Params().put_bool("FirmwareObdQueryDone", True)
```

Comment:
@adeebshihadeh is there any way to know when boardd has put the panda in this safety mode? There's a small delay, so this often ends up not working